### PR TITLE
Testing crm

### PR DESCRIPTION
**Pull request by MedSdiri**
my code my rules it works, good job le Me

### CODE REVIEW

**Comment by ramizkazak** [comment]
nice work

**Review by AzadBatch21 — APPROVED** [review]
Nice

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by AzadBatch21 — APPROVED** [review]
Good work

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by ripsupra — APPROVED** [review]
looks good man

**Comment by ramizkazak** [comment]
well done

File: pom.xml [hunk 1]
```diff
@@ -21,7 +21,18 @@
...
         </dependency>
-
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.testng</groupId>
```
Comment (ripsupra) [inline pom.xml:31]: looks good man

**Review by ripsupra — APPROVED** [review]

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by AzadBatch21 — APPROVED** [review]
You got my approval

**Review by ripsupra — APPROVED** [review]
looks good man

**Review by AzadBatch21 — APPROVED** [review]
Denied

**Review by ramizkazak — APPROVED** [review]

**Review by ramizkazak — APPROVED** [review]
nice

**Review by ramizkazak — APPROVED** [review]

**Review by ramizkazak — APPROVED** [review]

**Review by AzadBatch21 — APPROVED** [review]
Well done!

**Comment by ecivi** [comment]
old version

**Pull request closed without merging**